css/icons.scss

```diff
@@ -67,3 +67,23 @@
 		@include icon-color('star-dark', 'actions', 'FC0', 1, true);
 	}
 }
+
+// The atwho panel is a direct child of the body, so it is not affected by
+// .app-Talk rules above.
+.atwho-panel {
+	// "forced-white" needs to be included in the class name as the Avatar does
+	// not accept several classes.
+	.icon-group-forced-white {
+		background-image: url(icon-color-path('group', 'actions', 'fff', 1, true));
+	}
+
+	.atwho-view {
+		background: var(--color-main-background);
+		color: var(--color-main-text);
+	}
+
+	.atwho-cur {
+		background: var(--color-primary);
+		color: var(--color-primary-text);
+	}
```

Comment:
Either additional CSS files are allowed (even if it is just for corner cases) or they are not, but CSS rules unrelated to icons should not be sneaked in _icons.scss_.

Reply:
This can not be done in a vue  file because none of our vue files is a parent to the object and i would like to load as few files as possible. Loading a second file for just 2 rules sounds like madness to me